nipype/interfaces/spm/preprocess.py

```diff
@@ -843,8 +843,14 @@ class NewSegmentInputSpec(SPMCommandInputSpec):
                           field='tissue')
     affine_regularization = traits.Enum('mni', 'eastern', 'subj', 'none', field='warp.affreg',
                                         desc='mni, eastern, subj, none ')
-    warping_regularization = traits.Float(field='warp.reg',
-                                          desc='Aproximate distance between sampling points.')
+    warping_regularization = traits.Either(traits.Float(),
+                                           traits.List(traits.Float(),
+                                                       minlen=5, maxlen=5),
+                                           field='warp.reg'
+                                           desc='Warping regularization \
```

Comment:
instead of the `\` you can simply create a tuple

```
desc=('Warping regularization parameter(s). '
      'Accepts float or list of floats (the '
      'latter being required by SPM12)')
```